examples/server-demo/src/server.js

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+
+import App from './App';
+import html from './htmlTemplate';
+
+const PORT = process.env.PORT || 5657;
+const app = express();
+
+app.get('/', (req, res) => {
+  const app = ReactDOMServer.renderToString(<App />);
+
+  const initResults = {
+    recommendations: [
+      {
+        name: 'Sneakers Hogan Rebel pink',
+        objectID: 'MX1234',
+        price: {
+          value: 278,
+          currency: 'EUR',
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        image_urls: [
+          'https://res.cloudinary.com/hilnmyskv/image/upload/v1638372051/flagship_sunrise/M0E20000000DWIV_0.jpg',
+        ],
+      },
+      {
+        name: 'Sneakers Hogan Rebel multi',
+        objectID: 'MX1234',
+        price: {
+          value: 343,
+          currency: 'EUR',
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        image_urls: [
+          'https://res.cloudinary.com/hilnmyskv/image/upload/v1638372051/flagship_sunrise/M0E20000000DWIV_0.jpg',
+        ],
+      },
+      {
+        name: 'Sneakers Hogan Rebel white',
+        objectID: 'MX1234',
+        price: {
+          value: 268,
+          currency: 'EUR',
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        image_urls: [
+          'https://res.cloudinary.com/hilnmyskv/image/upload/v1638372051/flagship_sunrise/M0E20000000DWIV_0.jpg',
+        ],
+      },
+    ],
+  };
+
+  res.send(
+    html({
+      body: app,
+      initialState: initResults,
```

Comment:
Can we settle on a single name for consistency everywhere? The prop is named `initialState` so I would name the variable this way too.

Reply:
✅